benchmarks/micro/bench_orphan_lock_skip.py

```
@@ -0,0 +1,105 @@
+    # Warmup
+    for _ in range(10000):
+        old_check()
+        new_check()
+
+    t_old = timeit.timeit(old_check, number=n)
+    t_new = timeit.timeit(new_check, number=n)
+    ns_old = t_old / n * 1e9
+    ns_new = t_new / n * 1e9
+    saving = ns_old - ns_new
+    speedup = ns_old / ns_new if ns_new > 0 else float('inf')
+    print(f"  Empty orphaned set (common case):")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)